examples/preprocessing/plot_all_scaling.py

```diff
@@ -91,7 +93,9 @@
         QuantileTransformer(output_distribution='normal')
         .fit_transform(X)),
     ('Data after sample-wise L2 normalizing',
-        Normalizer().fit_transform(X))
+        Normalizer().fit_transform(X)),
+    ('Data after power transformation (Box-Cox)',
+     PowerTransformer(method='boxcox').fit_transform(X))
```

Comment:
can you please put this before QuantileTransformer, or after if you rather? It is a simple, parametric transform, and is quite unlike `Normalizer`. Perhaps put it before, but maybe switch the order of QuanitileTransformer(uniform) and QuanitileTransformer(normal) so those producing close-to-normal distributions are contiguous.

Reply:
Done